src/main/resources/anniv5Resources/localization/eng/monsterhunterpack/Keywordstrings.json

```
@@ -48,6 +48,6 @@
       "non-lethal",
       "non_lethal"
     ],
-    "DESCRIPTION": "Non-lethal damage will leave targets at 1 HP."
+    "DESCRIPTION": "Non-lethal damage will leave targets at 1 hp."
```

```suggestion
    "DESCRIPTION": "Non-lethal damage will leave targets at 1 HP."
```
??? Look at Bandage Up